WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/IssueRefundViewModelTests.swift

```diff
@@ -442,6 +442,37 @@ final class IssueRefundViewModelTests: XCTestCase {
         XCTAssertFalse(viewModel.isNextButtonEnabled)
     }
 
+    func test_viewModel_starts_with_no_saved_changes() {
```

Comment:
nit: would `no_unsaved_changes` be more consistent with the `hasUnsavedChanges` name?

```suggestion
    func test_viewModel_starts_with_no_unsaved_changes() {
```